File: hw/nase/u2/hw14 DBMongoandGUI/Python/model/mongo_connection.py
from pymongo import MongoClient

class MongoConnection:
    _client = None

    @classmethod
    def get_database(cls):
        if cls._client is None:
            uri = "mongodb+srv://Jennyfer:jennyfer@jennyfer.owlaicw.mongodb.net/?appName=Jennyfer"

            cls._client = MongoClient(uri)
        return cls._client["MothersAppDB"]

File: hw/nase/u2/hw14 DBMongoandGUI/JavaScript/model/mongoConnection.js
const { MongoClient } = require('mongodb');

class MongoConnection {
    static _client = null;

    static getDatabase() {
        if (!this._client) {
            const uri = "mongodb+srv://Jennyfer:jennyfer@jennyfer.owlaicw.mongodb.net/?appName=Jennyfer";
            
            this._client = new MongoClient(uri);
        }
        return this._client.db("MothersAppDB");
    }
}

module.exports = MongoConnection;